class/minio.yml

```diff
@@ -0,0 +1,29 @@
+parameters:
+  kapitan:
+    dependencies:
+      - type: helm
+        output_path: dependencies/minio/helmcharts/minio
+        source: https://charts.bitnami.com/bitnami
+        version: ${minio:charts:minio}
+        chart_name: minio
+    compile:
+      - input_paths:
+          - minio/component/app.jsonnet
+        input_type: jsonnet
+        output_path: apps/
+      - input_paths:
+          - minio/component/main.jsonnet
+        input_type: jsonnet
+        output_path: minio/
+      - input_paths:
+          - minio/component/ns.jsonnet
+        input_type: jsonnet
+        output_path: minio/
+      - output_path: minio/${_instance}/10_minio_helmchart
+        input_type: helm
+        output_type: yaml
+        input_paths:
+          - minio/helmcharts/minio
```

Comment:
Still needs

`helm_values: ${minio:helmValues}` to pass on the helm values configured in the hierarchy to the helm chart templater

Reply:
Fixed